lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -66,7 +66,7 @@ export const BrowserAuthErrorMessage = {
     },
     silentSSOInsufficientInfoError: {
         code: "silent_sso_error",
-        desc: "Silent SSO could not be completed - insufficient information was provided. Please provide either a login_hint, sid or account object."
```

Comment:
It does not look like sid or loginHint gets populated if you pass account, plus, the assertion in ssoSilent was only checking loginHint. Account support can be updated separately.